cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/index.js

```diff
@@ -0,0 +1,67 @@
+// import { render } from 'react-dom';
+
+// const App = () => (
+//     <section className="my-money-calendar">
+//      <div>hi, missy app</div>
+//     </section>
+// );
+
+// render(<App />, document.querySelector('#mmt-cost-of-credit-tool'));
+
+import * as idb from "idb";
+import { render } from "react-dom";
+import { Workbox } from "workbox-window";
+import { DateTime, Info } from "luxon";
```

Comment:
You probably don't need Luxon either

Reply:
I'm not sure about that, either.  I commented it out.  I just want to leave it as a reminder to evaluate it later.